test/passing/symbol.ml

```diff
@@ -0,0 +1,9 @@
+;;
+let op = if b then ( * ) else ( + ) in
+()
+
+;;
+assert ( * )
+
+;;
+(( * ) [@a])
```

Comment:
These parentheses are not necessary.
This could be fixed by checking `List.is_empty exp.pexp_attributes` in `Ast.is_displaced_infix_op`. Testing is needed to make sure this is sound.

Reply:
It improves this case indeed, thanks.